src/samples/FullPortal/index.tsx

```diff
@@ -136,7 +136,7 @@ export default function FullPortal() {
     else if( myLoadDefaultPortal ) {
       // eslint-disable-next-line no-console
       console.log(`Loading default portal`);
-      myLoadDefaultPortal();
+      myLoadDefaultPortal("pega-root", []);
```

Comment:
@vrseraphin  A few questions:

* Will this change break 8.7? (the main branch currently works for both 8.7 and 8.8)
* Will this change also be made to SDK-A and SDK-WC soon so they stay in sync?

Reply:
No.  As myLoadDefaultPortal will be undefined and this code path will not be traversed.  (so a capability check.  Will use only if there

Reply:
Yes, will get changes into other SDKs as well.